osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs

```diff
@@ -165,10 +168,11 @@ protected override DrawableHitObject CreateNestedHitObject(HitObject hitObject)
         }
 
         [BackgroundDependencyLoader]
-        private void load(OsuColour colours)
+        private void load(OsuColour colours, ISkinSource skin)
         {
             positionBindable.BindValueChanged(pos => Position = pos.NewValue);
             positionBindable.BindTo(HitObject.PositionBindable);
+            spinnerFrequencyModulate = skin.GetConfig<OsuSkinConfiguration, bool>(OsuSkinConfiguration.SpinnerFrequencyModulate)?.Value ?? true;
```

Comment:
This can't be done on `load()` as it'll break if you change between skins that have different values of this configuration. See `SkinReloadableDrawable.SkinChanged()` for how to do this properly.